flow-model-generator/modules/flow-model-generator-ls-extension/src/test/resources/search/config/connectors/database.json

```diff
@@ -168,7 +168,7 @@
     },
     {
       "metadata": {
-        "label": "Persist.redis RedisClient",
+        "label": "Redis RedisClient",
```

Comment:
isn't this confusing? :)

Reply:
Yes, we need to keep "persist" in the name. However, also thought of removing "persist" from the search since we're not supporting persistence in the GA.

Reply:
But in that case we'll end up displaying similar names for 2 modules right? (`ballerinax/persist.redis` and `ballerinax/redis`). 

we're not supporting persistence in the GA, one option would be to remove all the persist related modules from the palette and the search itself (and add after the GA)

Reply:
Can you check this one https://github.com/ballerina-platform/ballerina-dev-tools/blob/d9276210a93583eb4e64adb90ed8023eb915047d/flow-model-generator/modules/flow-model-generator-ls-extension/src/test/resources/search/config/connectors/persist.json?

Reply:
+1